base/loading.jl

```diff
@@ -341,7 +337,7 @@ during package development as code is modified.
 function reload(name::AbstractString)
     if isfile(name) || contains(name,Filesystem.path_separator)
         # for reload("path/file.jl") just ask for include instead
-        error("use `include` instead of `reload` to load source files")
+        error("Use `include` instead of `reload` to load source files.")
```

Comment:
If the error message is a complete sentence, it doesn't hurt to punctuate/capitalize it that way.